core/src/state.rs

```diff
@@ -348,13 +350,23 @@ impl Users1Data {
     pub fn account_id(&self) -> AccountId {
         match self {
             Self::V1(user) => user.account_id,
+            Self::V2(user) => user.account_id,
+        }
+    }
+
+    /// Latest attested user identity on radicle link.
+    pub fn link_user(&self) -> &Option<Bytes128> {
+        match self {
+            Self::V1(_) => &None,
+            Self::V2(user) => &user.link_user,
```

Comment:
How about this? It would make working with an API easier.
```suggestion
    pub fn link_user(&self) -> Option<&Bytes128> {
        match self {
            Self::V1(_) => None,
            Self::V2(user) => user.link_user.as_ref(),
```

Reply:
:facepalm: Done, thanks!